examples/ESP8266/GetObjectLikes/GetObjectLikes.ino

```diff
@@ -0,0 +1,131 @@
+/*********************************************************************
+ *  Get your Facebook friends count  *
```

Comment:
I've renamed the method and example name to better align with the FB API. My name is not needed in the description. Just happy to give back.